apps/biobin/resources/usr/bin/biobin-permute-collate.py

```diff
@@ -5,74 +5,66 @@
 
 
 if (len(sys.argv) < 3) or ("%d" not in sys.argv[2]):
-	print "usage: %s <baseline> <pathformat> [permutations]" % (sys.argv[0],)
-	print "\t<baseline> is the unpermuted summary.tsv file"
-	print "\t<pathformat> is the path to the permuted summary.tsv files,"
-	print "\t\twhich must contain \"%%d\" in place of the permutation number"
-	print "\t[permutations] is the number of permutations to scan; default=1000"
-	sys.exit(2)
+    print "usage: %s <baseline> <pathformat> [permutations]" % (sys.argv[0],)
+    print "\t<baseline> is the unpermuted summary.tsv file"
+    print "\t<pathformat> is the path to the permuted summary.tsv files,"
+    print "\t\twhich must contain \"%%d\" in place of the permutation number"
+    print "\t[permutations] is the number of permutations to scan; default=1000"
+    sys.exit(2)
 
 
 pMin = 1
```

Comment:
Again, minor issue, but I would set pMin = 0 here, and on line 21 do pmin = int(pRange[0]) - 1, and on line 45 do range(pMin, pMax).